tools/coreml/test/test_mxnet_converter.py

```diff
@@ -445,7 +445,7 @@ def test_tiny_conv_random_input_multi_filter(self):
         self._test_mxnet_model(net, input_shape=input_shape, mode='random')
 
     def test_conv_random(self):
-        np.random.seed(1988)
+        np.random.seed(1986)
```

Comment:
wondering why you changed the seed value here from 1988 to 1986, is there any significance to this number?